shelley/chain-and-ledger/executable-spec/test/Examples.hs

```diff
@@ -108,9 +116,55 @@ aliceInitCoin = 10000
 bobInitCoin :: Coin
 bobInitCoin = 1000
 
+alicePoolParams :: PoolParams
+alicePoolParams =
+  PoolParams
+    { _poolPubKey = vKey aliceOperator
+    , _poolPledge = Coin 1
+    , _poolCost = Coin 5
+    , _poolMargin = unsafeMkUnitInterval 0.1
+    , _poolRAcnt = RewardAcnt aliceSHK
+    , _poolOwners = Set.singleton $ (hashKey . vKey) aliceStake
+    }
+
+
+-- | Helper Functions
+
+
+mkBlock :: Maybe HashHeader -> KeyPair -> (SKeyES, VKeyES) -> [Tx] -> Slot -> Seed -> Seed
+  -> UnitInterval -> Natural -> Block
+mkBlock prev cold (shot, vhot) txns s enonce bnonce l kesPeriod =
+  let
+    bhb = BHBody
+            prev
+            (vKey cold)
+            s
+            bnonce
+            (Proof (vKey cold) enonce)
+            l
+            (Proof (vKey cold) l)
+            (fromIntegral $ bBodySize txns)
+            (bhbHash [txEx2])
+            (OCert
+              vhot
+              (vKey cold)
+              0
+              (KESPeriod 0)
+              (sign (sKey cold) (vhot, 0, KESPeriod 0))
+            )
+            (ProtVer 0 0 0)
+    bh = BHeader bhb (Keys.signKES shot bhb kesPeriod)
+  in
+    Block bh txns
+
+unsafeMkUnitInterval :: Rational -> UnitInterval
```

Comment:
:-)
would it make sense to move this into `BaseTypes` ?

Reply:
maybe? or keep it just in testing somehow?